tests/conftest.py

```diff
@@ -344,10 +379,19 @@ def verification_params(address):
 
 
 @pytest.fixture(scope="session")
-def address_to_verify(address, project):
-    contract_type = project.get_contract("foo").contract_type
-    ape.chain.contracts._local_contract_types[address] = contract_type
-    return address
+def address_to_verify(fake_connection, project, account):
+    # Deploy the library first.
+    library = account.deploy(project.MyLib)
+    ape.chain.contracts._local_contract_types[library.address] = library.contract_type
+
+    # Add the library to recompile contract `foo`.
+    solidity = project.compiler_manager.solidity
+    solidity.add_library(library)
+
+    # Use foo address for verification.
+    foo = project.foo.deploy(sender=account)
```

Comment:
it looks like this isn't compiling properly: 
`E       AttributeError: ProjectManager has no attribute or contract named 'foo'.`

Reply:
when I run pytest locally it looks like `mylib` is not have an attribute: 
```
E       AttributeError: ProjectManager has no attribute or contract named 'MyLib'.
```

Reply:
```
$ ape compile -f
INFO: Compiling 'foo.sol'.
WARNING: Libraries must be deployed and configured separately.
```

```
In [1]: accounts.test_accounts[0].deploy(project.foo)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In [1], line 1
----> 1 accounts.test_accounts[0].deploy(project.foo)

File ~/ape/ape/src/ape/managers/project/manager.py:447, in ProjectManager.__getattr__(self, attr_name)
    444 contract_type = contract_types.get(attr_name)
    445 if not contract_type:
    446     # Still not found. Contract likely doesn't exist.
--> 447     return self._handle_attr_not_found(attr_name)
    449 result = self._get_attr(attr_name)
    450 if not result:
    451     # Shouldn't happen.

File ~/ape/ape/src/ape/managers/project/manager.py:499, in ProjectManager._handle_attr_not_found(self, attr_name)
    493 if missing_exts:
    494     message = (
    495         f"{message} Could it be from one of the missing compilers for extensions: "
    496         + f'{", ".join(sorted(missing_exts))}?'
    497     )
--> 499 raise AttributeError(message)

AttributeError: ProjectManager has no attribute or contract named 'foo'.
```

Reply:
```
In [6]: project.contracts.items()
Out[6]: dict_items([('MyLib', <ContractType MyLib>)])
```
so it looks like MyLib is there in the contract items but foo is not.

Reply:
The issue is fixed here: https://github.com/ApeWorX/ape/pull/1341